ProgressBar/tests/ProgressBar-specs.js

```diff
@@ -1,28 +1,29 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import ProgressBar from '../ProgressBar';
 
 describe('ProgressBar Specs', () => {
 	test('should only show tooltip when tooltip is true', () => {
-		const progressBar = mount(
+		render(
 			<ProgressBar tooltip />
 		);
+		const progressBar = screen.getByRole('progressbar').children.item(1);
+		const expected = 'tooltip';
 
-		const expected = 1;
-		const actual = progressBar.find('ProgressBarTooltip').length;
-
-		expect(actual).toBe(expected);
+		expect(progressBar).toHaveClass(expected);
 	});
 
 	test('should have tooltip show progress as percentage', () => {
-		const progressBar = mount(
+		render(
 			<ProgressBar
```

Comment:
Please sort props. Also, you can put the renders on a single line (above as well).